src/utils/starknet_compilation.py

```diff
@@ -57,11 +57,8 @@ def preprocess_contract(
             return context.preprocessed_program
         except FileNotFoundError as err:
             raise StarknetCompiler.NotExistingMainFileException(
-                message=(
-                    f"Couldn't find a contract '{err.filename}'\n"
-                    'Did you forget to update protostar.toml::["protostar.contracts"]?'
-                )
-            )
+                message=(f"Couldn't find a contract '{err.filename}'")
```

Comment:
isn't this error going to be raised when a single file is missing? 
i.e. 

`signer = ["interface_1.cairo", "interface_2.cairo"]` -  when `interface_2.cairo` is missing, the message can be misleading (a part of the interface cannot be found, not a "contract" per se)